ndmg/register/gen_reg.py

```diff
@@ -115,23 +95,37 @@ def direct_streamline_norm(streams, fa_path, namer):
     trk_hdr['image_orientation_patient'] = np.array([0., 0., 0., 0., 0., 0.]).astype('float32')
     trk_hdr['endianness'] = '<'
     trk_hdr['_offset_data'] = 1000
-    trk_hdr['nb_streamlines'] = len(streamlines)
-    tractogram = nib.streamlines.Tractogram(streamlines, affine_to_rasmm=trk_affine)
+    trk_hdr['nb_streamlines'] = len(mni_streamlines)
+    tractogram = nib.streamlines.Tractogram(mni_streamlines, affine_to_rasmm=trk_affine)
     trkfile = nib.streamlines.trk.TrkFile(tractogram, header=trk_hdr)
-    nib.streamlines.save(trkfile, streams)
+    streams_mni = streams.split('.trk')[0] + '_dsn.trk'
+    nib.streamlines.save(trkfile, streams_mni)
 
     # DSN QC plotting
     mgu.show_template_bundles(mni_streamlines, template_path, streams_warp_png)
 
-    return mni_streamlines
+    return mni_streamlines, streams_mni
 
 
 class dmri_reg(object):
     def __init__(self, namer, nodif_B0, nodif_B0_mask, t1w_in, vox_size, simple):
+        import os.path as op
+        if os.path.isdir("/ndmg_atlases"):
```

Comment:
maybe we should just make a `get_atlas_dir` function?

Reply:
I love that idea. Feel free to create a prototype!

Reply:
> I love that idea. Feel free to create a prototype!

will put it on my growing list of minor stuff to do (although, pretty low-priority I think)